backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -675,4 +675,16 @@ class TestOperator extends VeloxWholeStageTransformerSuite with AdaptiveSparkPla
       }
     }
   }
+
+  test("Verify custom tokenizer") {
+    // https://github.com/apache/spark/pull/35229 Spark remove parquet column name check after 3.2
+    if (!spark.version.contains("3.2")) {
```

Comment:
Maybe, better to utilize a spark function `majorMinorVersion` to get the exact major/minor version to check.

Reply:
Maybe, use field name with special char for meeting the test purpose?

Reply:
ShimDescriptor does not expose `toMajorMinorVersion`.